freqtrade/freqtradebot.py

```diff
@@ -362,13 +364,56 @@ def process_maybe_execute_sell(self, trade: Trade) -> bool:
         if trade.open_order_id:
             # Update trade with order values
             logger.info('Found open order for %s', trade)
-            trade.update(exchange.get_order(trade.open_order_id, trade.pair))
+            order = exchange.get_order(trade.open_order_id, trade.pair)
+            # Try update amount (binance-fix)
+            try:
+                # Only run for closed trades
```

Comment:
The comment is misleading, because the code only runs for closed orders not trades